R/NBthDE.R

```diff
@@ -322,171 +543,15 @@ setMethod(
 #'
 #' @rdname fitNBthDE-methods
 #' @aliases fitNBthDE,matrix-method
```

Comment:
add roxygen alias for dgcmatrix

Reply:
This still needs attention

Reply:
@NicoleEO  Added the method for dgcmatrix in my branch will be pushing it once I have addressed others

Reply:
@AMAKIRI-AUGUSTINE, I was able to get the result I wanted by just using @rdname to put in the right markdown file and @export to ensure the method gets into the markdown.
![image](https://user-images.githubusercontent.com/20404635/155432392-d3dedbf4-aa2a-4c1d-a43f-57bb2b176851.png)